src/bun.js/bindings/bindings.zig

```diff
@@ -2414,6 +2414,19 @@ pub const JSGlobalObject = extern struct {
         return err;
     }
 
+    pub fn createError(
+        this: *JSGlobalObject,
+        comptime code: JSC.Node.ErrorCode,
```

Comment:
only `message` and `args` needs to be comptime

Reply:
👍